static/admin/config.yml

```diff
@@ -37,10 +37,30 @@ collections: # A list of collections the CMS should be able to edit
         file: "content/copy/home.yml"
         description: "Copy for the home page"
         fields:
+          - name: callout
+            label: "Left Callout"
+            widget: object
+            fields:
+
+                - name: title
+                  label: Title
+                  widget: string
+                  tagname: h1
+
+                - name: paragraph
+                  label: Paragraph
```

Comment:
Instead of paragraph let's call this copy

Reply:
I think paragraph is more descriptive for the copy editor. It tells them that the copy should be structured in sentence format and limited to 1 paragraph.

Reply:
I've never seen that, though. How about `body`?